File: 코딩 기초 트레이닝/세로 읽기-p181904/세로 읽기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181904

function solution(my_string, m, c) {
    var answer = '';
    my_string = my_string.split("");

    let cnt = 0;

    my_string.forEach((d) => {
        cnt += 1;
        if(cnt === c) {
            answer += d
        };

        if(cnt === m) {
            cnt = 0;
        }
    });

    return answer;
}

File: 코딩 기초 트레이닝/세로 읽기-p181904/세로 읽기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181904

def solution(my_string, m, c):
    answer = ''
    my_string = list(my_string)
    cnt = 0

    for idx, v in enumerate(my_string):
        cnt += 1
        if cnt == c:
            answer += v

        if cnt == m:
            cnt = 0

    return answer
